docs/docs/frequently-asked-questions.md

```diff
@@ -66,22 +66,15 @@ Validators can also be representatives but representatives are not required to r
 
 This feature discerns between validator operators (infrastructure) and governance representatives which further democratizes the ecosystem. This also reduces the pressure on validators to be involved in on-chain governance.
 
-## Can validators opt-out of replicated security?
+## Can validators opt out of replicated security?
```

Comment:
```suggestion
## Can validators opt out of validating a consumer chain?
```

Reply:
Replicated security is specifically full security, so by our definitions, you can never opt out of replicated security